### ISSUE TITLE
Show more characters in resource names

### ISSUE THREAD

**Issue by chris-armstrong** [aws-solutions/workload-discovery-on-aws#550]
**Feature name**
Show more characters in resource names

**Is your feature request related to a problem? Please describe.**
I am trying to visualise serverless-style deployments that have a lot of resources, often with autogenerated names or names with fixed prefixes. The 13 character limit on names before they are truncated e.g. `honey-shared-application` -> `honey-shared...` makes it hard to distinguish resources.

**Describe the feature you'd like to see implemented**
* Show resources with longer names without truncation (and more diagram spacing to separate them)
* strip out prefixes common to numerous resources or shorten them in the diagram

**Describe the value this feature will add to AWS Perspective**
Enable lots of resources with similar prefixes or long names to be distinguished in a diagram.

**Describe alternatives you've considered**
I could redeploy resources with shorter names, but this is a hassle in order to visualise them automatically.

**Additional context**
Many of our resources are prefixed with stack names or environment names.

**Comment by svozza** [comment]
My concern here is the amount of extra whitespace having the full names would introduce into diagrams. I do wonder though if we could make this configurable in the **Diagram Settings** section of the diagram so user could choose.

**Comment by chris-armstrong** [comment]
I fully agree, and think that multiple solutions may needed, because some users may have very short names (although the current 12 characters is _very_ short). However, at the moment the tool is virtually unusable for my use case because I can't configure anything to either select part of the names, filter out common parts, or show more of the names.

**Comment by svozza** [comment]
We've added this feature request to our backlog.

**Comment by mattspradley** [comment]
Adding  on hover support to show the full name would be useful.
